.github/workflows/tools-buildtest.yml

```diff
@@ -9,6 +9,7 @@ on:
   pull_request:
     branches:
       - master
+      - [0-9][0-9][0-9][0-9].[0-9][0-9]-branch
```

Comment:
I just wonder, why this one doesn't have single quotes like the other one?

Reply:
I amended this ...

Reply:
(but Github does not show it :()

Reply:
Now its there!